plugins/repository-azure/licenses/azure-LICENSE.txt

```diff
@@ -0,0 +1,1565 @@
+
+
+
+<!DOCTYPE html>
```

Comment:
I see this hasn't changed but...is this really the license? Looks like garbage html, not a txt license as the file suffix suggests...